firebase-messaging.html

```diff
@@ -134,7 +140,7 @@
            */
           token: {
             type: String,
-            value: null,
+            value: "",
```

Comment:
This is a breaking change. The Polymer -> Closure compiler translates `String` into `string`. This means in the definition here, you should specify `@type {String}` which is nullable.